labelCloud/view/gui.py

```diff
@@ -658,3 +671,30 @@ def change_label_color(self):
         LabelConfig().set_class_color(
             bbox.classname, Color3f.from_qcolor(QColorDialog.getColor())
         )
+
+    @staticmethod
+    def save_point_cloud_as(pointcloud: PointCloud) -> None:
+        extensions = BasePointCloudHandler.get_supported_extensions()
+        make_filter = " ".join(["*" + extension for extension in extensions])
+        file_filter = f"Point Cloud File ({make_filter})"
+        file_name, _ = QFileDialog.getSaveFileName(
+            caption="Select a file name to save the point cloud",
+            directory=str(pointcloud.path.parent),
+            filter=file_filter,
+            initialFilter=file_filter,
+        )
+        if file_name != "":
```

Comment:
Here you could also pull the error case forward:

```python

if not file_name:
  # log warning or open dialog
  return

...

try:
  ...
except SaveException as exc:
  ...
```

Reply:
changed here [62b797c](https://github.com/ch-sa/labelCloud/pull/129/commits/62b797ce6ad013a5cc245d2ab1d7aa0b3c235dd5)